lib/bob/job/docker_checker.ex

```diff
@@ -186,7 +186,12 @@ defmodule Bob.Job.DockerChecker do
   defp build_elixir_ref?("v0." <> _), do: false
 
   defp build_elixir_ref?("v" <> version) do
-    match?({:ok, %Version{pre: []}}, Version.parse(version))
+    case Version.parse(version) do
+      # don't build RCs for < 1.12
```

Comment:
I think we should build for all versions but we can start with this if it means we can get it out quicker.

Reply:
Sounds good, I'll do this change soon.